lib/widgets/content.dart

```diff
@@ -687,6 +692,41 @@ class UserMention extends StatelessWidget {
 //     borderRadius: BorderRadius.all(Radius.circular(3))));
 }
 
+class GlobalTime extends StatelessWidget {
```

Comment:
nit: put this after MessageImageEmoji, to match the if/else-if chain above (which matches the node-class definitions)